config_cmd.go

```diff
@@ -115,6 +117,20 @@ template      | The path to a feed-specific email template to use.
 user-agent    | Configure a specific User-Agent when making HTTP requests.
 
 
+Polling Frequency
+-----------------
+
+Assuming you have a feed you only want to check once an hour, with a default
+sleep-delay of fifteen minutes the deamon command would poll the feed too often.
+
+However if you changed the sleep-delay to 60 then other feeds which you want to
+track more closely would get delayed notifications.
+
+The "frequency" argument tracks the time between the last fetch of a feed, and
+allows you to say "Fetch this feed only if it was fetched >XX minutes ago", even
+if the sleep-delay is set to something short like a minute.
+
+
```

Comment:
IIUC the frequency cannot be higher than `sleep-delay`. For instance I cannot set `sleep-delay = 15m` and `frequency = 5m` for a specific feed.

Maybe it is worth mentioning here? Just in case it is not clear to new users.

Reply:
If I remove the sleep delay this point becomes moot :)

Reply:
But I think the actual behaviour is the other way around to your comment:

* sleep-delay would contain the smallest amount of time a feed could be polled.

If sleep delay were 30 and frequency 5 then the feed would be notified at most once every 30 minutes.  Since during sleep-delay nothing happens.

I liked your comment about removing the sleep delay a little, but on reflection I guess removing it would lead to:

1. Startup
2. For each feed
   * Fetch feed.
   * Email if necessary.
3. Goto 2

Without the sleep delay we'd be spinning in our loop very quickly.  Iterating over ever feed in a tight loop.  It might be that I should just ignore the users' setting and make the sleep delay always five minutes.  That would then be the smallest possible `frequency` setting.

So:

* Make the daemon command ALWAYS sleep 5 minutes.
* Use `SLEEP` to set the default `frequency` if not changed on a per-feed basis.

That way if a user has currently set SLEEP to be 15/30 it would take effect, but could still be altered.

Reply:
I think I've made the change now:

* Sleep 5 minutes always, between feed-loops.
* If SLEEP is set it becomes the default feed-frequency.
* But the per-feed option will override that as necessary.

Reply:
Cool. So, please correct me if I understood wrong:

- `SLEEP` will be used to set the default `frequency` for feeds that do not override it with a per-feed `frequency` option.
- The new daemon behavior is to sleep 5 minutes between each run and this cannot be configured.
- If a per-feed `frequency` option is specified this takes precedence.
- If a per-feed `frequency` option is lower than 5 minutes, it won't be honored because there is a sleep of 5 minutes after each run. 
    - Actually, from feeds point of view it would be 5 minutes + the amount of time the script takes to execute.

Another way could be to instead of statically sleep 5m, you sleep 5m by default and if you encounter one or more per-feed `frequency` lower than that... then you adjust the sleep delay to the lowest `frequency` found.

Reply:
It seems like your understanding is correct.

I think that polling a feed for less than five minutes isn't something that I want to support,  so I'd probably not drop the threshold below that.